deploy/examples/subvolumegroup.yaml

```
@@ -6,6 +6,14 @@ metadata:
   namespace: rook-ceph # namespace:cluster
 spec:
   # The name of the subvolume group. If not set, the default is the name of the subvolumeGroup CR.
-  # name: group-a
+  # name: group-a1
   # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
   filesystemName: myfs
+  # reference https://docs.ceph.com/en/latest/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+  # only one out of (export, distributed, random) can be set at a time
+  # by default pinning is set with default values(distributed=1)
+  # for disabling default values set(distributed=0)
```

In doc text, there should be a space between a word and a parenthetical

```suggestion
  # by default pinning is set with default values (distributed=1)
  # for disabling default values set (distributed=0)
```